src/interactive/AccordionPanel.jsx

```diff
@@ -34,42 +32,41 @@ class AccordionPanel extends React.Component {
 	}
 
 	/**
+	 * 
+	 * @description calls the AccordionPanelGroups's callback to toggle open state
+	 * and render the `AccordionPanel` open or closed, sets height in state
+	 * @param {Event} e - the event object
 	 * @returns {undefined}
-	 *
-	 * Updates state to toggle `AccordionPanel` open and closed
 	 */
-	_handleToggle(){
-		if (this.props.setClickedPanel) {
-			this.props.setClickedPanel(this);
-		}
+	_handleToggle(e){
+		e.preventDefault();
+		e.stopPropagation();
```

Comment:
I think we just want to `preventDefault` here - unless there's a reason we're also stopping propagation?

Reply:
I need to try it in mup-web and make sure the form doesnt attempt submission.